cmd/create/cluster/cmd.go

```diff
@@ -3017,6 +3077,55 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 }
 
+func GetBillingAccountContracts(cloudAccounts []*accountsv1.CloudAccount,
+	billingAccount string) ([]*accountsv1.Contract, bool) {
+	var contracts []*accountsv1.Contract
+	var isContractEnabled bool
+	for _, account := range cloudAccounts {
+		if account.CloudAccountID() == billingAccount {
+			contracts = account.Contracts()
+			for _, contract := range contracts {
+				for _, dimension := range contract.Dimensions() {
+					if ocm.HasValidContracts(dimension) {
+						isContractEnabled = true
+						break
+					}
+				}
+			}
+
+		}
+	}
+	return contracts, isContractEnabled
+}
+
+func PrintContracts(contracts []*accountsv1.Contract, r *rosa.Runtime) {
+	for _, contract := range contracts {
+		format := "Jan 02, 2006"
+		dimensions := contract.Dimensions()
+		var prePurchaseInfo string
+
+		for _, dimension := range dimensions {
+			numberOfVPCs := ocm.GetDimensionValue(dimension, "four_vcpu_hour")
+			numberOfClusters := ocm.GetDimensionValue(dimension, "control_plane")
+			prePurchaseInfo = prePurchaseInfo + fmt.Sprintf("   | Number of vCPUs:    |'%s'             | \n"+
```

Comment:
prePurchaseInfo will be empty at this point - probably safe to remove it from the right side of the assignment.

Reply:
@cristianoveiga The prePurchaseInfo will not be empty if there are multiple dimensions though. I recall that we would need to accommodate the scenario where there could be multiple dimensions and multiple contracts.